tests/csm/rest/test_csm_users.py

```diff
@@ -4655,6 +4655,117 @@ def test_28516(self):
         self.csm_obj.check_expected_response(response, HTTPStatus.OK)
 
 
+    @pytest.mark.sanity
```

Comment:
Is this sanity test?

Reply:
thought it should be

Reply:
If this is not part of CICD sanity as I can see its getting skipped as well, lets not add label as we use marker based sanity run for feature sanity.